Makefile

```diff
@@ -236,16 +234,16 @@ docker-build:
 	@echo "*** DEPRECATED: Use the image-hive target instead ***"
 	$(DOCKER_CMD) build -t ${IMG} .
 
-# Build the dev image using docker
-.PHONY: docker-dev-build
-docker-dev-build: build
-	@echo "*** DEPRECATED: Use the image-hive-dev target instead ***"
-	$(DOCKER_CMD) build -t ${IMG} -f Dockerfile.dev .
+.PHONY: buildah-makecache
+buildah-makecache:
+	mkdir -p /tmp/centos8/dnf
+	podman run --rm -v /tmp/centos8/dnf:/var/cache/dnf:z centos:8 dnf -y makecache
 
-# Build the dev image using builah
-.PHONY: buildah-dev-build
-buildah-dev-build:
-	buildah bud -f Dockerfile --tag ${IMG}
+# Build the dev image using buildah
+.PHONY: buildah-build
+buildah-build:
+	# Run buildah build with an overlay
+	buildah bud -v ~/.cache/go-build:/.cache/go-build:O -v /tmp/centos8/dnf:/var/cache/dnf:O -f Dockerfile --tag ${IMG}
```

Comment:
you can get go cache location using `go env GOCACHE`

Reply:
This was a mistaken commit from attempts to get the go cache mounted in, removed now.